extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -62,53 +70,43 @@ function ScrollerWithRef(
 ) {
   // We still need our own ref that we can always rely on to be there.
   const containerRef = useRef(null);
+  const axis = useMemo(() => Axis.X, []);
```

Comment:
Why `useMemo()` here?

Reply:
Whoops, I meant `useState` 😬  - in the PR where we introduce `horizontal` prop we will want to update that state accordingly.